Exiled.Events/Config.cs

```diff
@@ -22,13 +22,13 @@ public sealed class Config : IConfig
         /// Gets or sets a value indicating whether SCP-173 can be blocked or not by the tutorial.
         /// </summary>
         [Description("Indicates whether SCP-173 can be blocked or not by the tutorial")]
-        public bool CanTutorialBlockScp173 { get; set; } = true;
```

Comment:
Both of these should be true to emulate base game behavior, unless this was recently changes by Northwood.

Reply:
Tutorial blocks SCP-173 as well as he triggers SCP-096 on a server with no EXILED installed. So I think my changes are correct and will emulate base game behaviour.

Reply:
Re-read the names of those configs and try again. If these are both true, it means tutorials can stop 173 and trigger 096.

Reply:
Oh, okay my bad.